lib/battle/CBattleInfoEssentials.h

```diff
@@ -76,15 +76,15 @@ class DLL_LINKAGE CBattleInfoEssentials : public virtual CCallbackBase, public I
 	si8 battleTacticDist() const override; //returns tactic distance in current tactics phase; 0 if not in tactics phase
 	si8 battleGetTacticsSide() const override; //returns which side is in tactics phase, undefined if none (?)
 
-	bool battleCanFlee(PlayerColor player) const;
-	bool battleCanSurrender(PlayerColor player) const;
+	bool battleCanFlee(const PlayerColor & player) const;
+	bool battleCanSurrender(const PlayerColor & player) const;
 
-	ui8 otherSide(ui8 side) const;
-	PlayerColor otherPlayer(PlayerColor player) const;
+	static ui8 otherSide(ui8 side);

```

Comment:
Please revert. Callbacks should only use access through instance.

Reply:
Do not understand your static hate, but done(